client/src/views/Graphs/components/DrilldownPanel/EdgePane.vue

```diff
@@ -57,40 +76,73 @@
       this.fetchExternalData();
     }
 
+    get hasEdgeEvidences (): boolean {
+      return this.edgeEvidences.length > 0;
+    }
+
     async fetchExternalData (): Promise<void> {
-      this.externalData = await emmaaEvidence({
+      this.isLoading = true;
+      this.edgeEvidences = [];
+
+      const resultEmmaaEvidence = await emmaaEvidence({
         stmt_hash: this.data.statement_id,
         source: 'model_statement',
         model: this.model,
         format: 'json',
       });
+
+      if (resultEmmaaEvidence.evidence) {
+        this.edgeEvidences = await Promise.all(resultEmmaaEvidence.evidence.map(async (evidence) => {
+          const args = { doi: evidence.text_refs.DOI };
+          const artifact = await cosmosArtifactsMem(args);
+          return { artifact, evidence };
+        })) as EdgeEvidence[];
+      }
+
+      this.isLoading = false;
+    }
+
+    caption (artifact: COSMOS.CosmosArtifactInterface): string {
+      if (artifact.bibjson) {
+        const title = truncateString(artifact.bibjson.title, 50);
+        const authorList = truncateString(getAuthorList(artifact.bibjson), 50);
+        const timestamp = artifact.bibjson?.year?.toString();
+        return [title, authorList, timestamp].filter(Boolean).join(' - ');
+      }
     }
 
-    get isEmptyMetadata (): boolean {
-      return _.isEmpty(this.data);
+    excerpt (evidence: EMMAA.EmmaaEvidenceEvidenceInterface): string {
+      return truncateString(evidence.text, 500);
     }
 
-    onClickEvidence (evidence: EmmaaEvidenceEvidenceInterface): void {
+    onClickEvidence (evidence: EMMAA.EmmaaEvidenceEvidenceInterface): void {
       this.$emit('evidence-click', evidence.text_refs.DOI);
     }
   }
 </script>
 
 <style scoped>
-  ul {
-    list-style-type: none;
-    margin-left: 0;
-    padding-left: 0;
-  }
-
-  li {
+  figure {
     border: var(--border);
     cursor: pointer;
-    padding: 4px 8px;
+    font-size: .9em;
+    margin: .5em 0;
+    padding: .5rem;
+  }
+
+  figure:hover {
+    border-color: var(--selection);
```

Comment:
I think the yellow doesn't work that great with the white background, not enough contrast. This should be probably `--selection-dark`.

Reply:
![Screen Shot 2021-07-09 at 10 18 49](https://user-images.githubusercontent.com/636801/125092216-1b74e680-e09f-11eb-9025-6855e76b3faa.png)